scripts/bridges/pack_data.js

```diff
@@ -0,0 +1,39 @@
+/*global process*/
+
+const { ethers } = require("ethers");
+
+async function main() {
```

Comment:
Packing data script to prepare cross-bridge transaction for L2 that has to be executed by the Timelock